pyproject.toml

```diff
@@ -123,7 +121,6 @@ namespaces = false
     '*.png',
     '*.gif'
 ]
-'tests' = ['*']
```

Comment:
We no longer include the tests inside our builds, this is why I removed this line